pdal/GDALUtils.cpp

```diff
@@ -195,6 +195,18 @@ bool reprojectBounds(BOX3D& box, const std::string& srcSrs,
 }
 
 
+bool reprojectBounds(Bounds& box, const std::string& srcSrs,
```

Comment:
I don't understand this.  You shouldn't treat Bounds as BOX3D.  It also doesn't seem to be used.